python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -269,6 +272,13 @@ def constructExtraFiles(self, job):
 
         folder_dir = master_job.getInputWorkspace(create=True).getPath()
 
+        extraopts = ' '
```

Comment:
I don't think it wants to be the default because that implies that anything the user enters would overwrite them - these extra lines have to be added for the summary.xml to be generated so we can get the metadata.